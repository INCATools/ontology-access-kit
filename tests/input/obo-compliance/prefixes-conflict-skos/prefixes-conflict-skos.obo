format-version: 1.4
ontology: prefixes-conflict-skos
idspace: skos http://example.org/not-skos/

[Term]
id: X:1
property_value: skos:exactMatch Y:1

[Typedef]
id: skos:exactMatch
is_metadata_tag: true

