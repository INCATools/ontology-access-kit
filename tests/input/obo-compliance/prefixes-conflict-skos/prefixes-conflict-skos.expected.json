{
  "graphs" : [ {
    "id" : "http://purl.obolibrary.org/obo/prefixes-conflict-skos.owl",
    "meta" : {
      "basicPropertyValues" : [ {
        "pred" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
        "val" : "1.4"
      } ]
    },
    "nodes" : [ {
      "id" : "http://example.org/not-skos/exactMatch",
      "type" : "PROPERTY",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://www.geneontology.org/formats/oboInOwl#is_metadata_tag",
          "val" : "true"
        } ]
      }
    }, {
      "id" : "http://purl.obolibrary.org/obo/X_1",
      "type" : "CLASS",
      "meta" : {
        "basicPropertyValues" : [ {
          "pred" : "http://example.org/not-skos/exactMatch",
          "val" : "http://purl.obolibrary.org/obo/Y_1"
        } ]
      }
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#hasOBOFormatVersion",
      "lbl" : "has_obo_format_version",
      "type" : "PROPERTY"
    }, {
      "id" : "http://www.geneontology.org/formats/oboInOwl#id",
      "lbl" : "id",
      "type" : "PROPERTY"
    } ]
  } ]
}